src/Microsoft.ML.EntryPoints/ModelOperations.cs

```diff
@@ -2,16 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.ML.Runtime;
-using Microsoft.ML.Runtime.CommandLine;
-using Microsoft.ML.Runtime.Data;
-using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML;
+using Microsoft.ML.CommandLine;
+using Microsoft.ML.Data;
+using Microsoft.ML.EntryPoints;
+using Microsoft.ML.Learners;
 using Microsoft.ML.Trainers;
```

Comment:
Seems odd that we'd have `Microsoft.ML.Trainers` & `Microsoft.ML.Learners`, given the similar naming.

Also, it's not clear to me why `Microsoft.ML.Learners` was added to the list.

Was `Trainers` partially renamed to `Learners` and now we have both? _(rampant speculation)_

Reply:
This change does not rename Trainers to Learners. It just removes runtime from all namespaces.